x/distribution/legacy/v036/types.go

```diff
@@ -13,6 +19,12 @@ import (
 
 const (
 	ModuleName = "distribution"
+
+	// RouterKey is the message route for distribution
+	RouterKey = ModuleName
+
+	// ProposalTypeCommunityPoolSpend defines the type for a CommunityPoolSpendProposal
+	ProposalTypeCommunityPoolSpend = "CommunityPoolSpend"
 )
```

Comment:
why don't we import this from `x/distribution/types/keys.go`?

Reply:
Bez preferred to have migration logic self-contained and not rely on the module itself, ref: https://github.com/cosmos/cosmos-sdk/pull/6839#discussion_r465123712.

Note that we're breaking the above rule for >=0.40, as we're now using proto types from the module for 0.40.